### ISSUE TITLE
RFE: fn init should use v0.0.0 when bootstrapping a function

### ISSUE THREAD

**Issue by delabassee** [fnproject/cli#498]
`fn init` generates a v0.0.1 of a function, regardless of the language. This version will then be bumped to 0.0.2 during the first deployment. It would be more logical that `fn init` starts at v0.0.0 instead, the version would then be bumped to v0.0.1 during the initial deployment.

**Comment by riconnon** [comment]
This is likely to turn into another holy war, but personally I dislike the idea of monotonic versioning of functions by the CLI. I'd rather see it taken away in its entirety.
When two different people modify the function code on two different branches and run `fn deploy` we'll now have two people who believe that their function is version 0.0.x (where x is whatever's on master + 1)

**Comment by rdallman** [comment]
> This is likely to turn into another holy war, but personally I dislike the idea of monotonic versioning of functions by the CLI. I'd rather see it taken away in its entirety.

nope, use vcs. this isn't a productive conversation, we need versioning, semver is fine, and we need to better integrate it, but taking out the postmodern hammer while fun isn't productive here - versioning is a requirement and we need to do it for users, they won't do it themselves, happy to talk about improvements (like function versioning instead of image versioning) but not throwing out babies and bath water.

**Comment by rdallman** [comment]
if 2 users want to test 'dev' versions of the function the answer at the moment is likely 'copy it to DevFunctionXxx' and have a 'ProdFunctionXxx' and set up some schema like that. we can talk about how to integrate that better for users and if we should be managing that for users
